include/llvm-c/Linker.h

```diff
@@ -34,6 +34,12 @@ typedef enum {
 */
 LLVMBool LLVMLinkModules2(LLVMModuleRef Dest, LLVMModuleRef Src);
 
+LLVMLinkerCtx LLVMGetLinkerCtx(LLVMModuleRef Dest) ;
+
+LLVMBool LLVMLinkModuleToCtx(LLVMLinkerCtx Dest, LLVMModuleRef Src) ;
+
+void LLVMLinkerCtxDispose(LLVMLinkerCtx L) ;
```

Comment:
Just in terms of popularity, I think there are more existing `LLVMDisposeFoo` functions than `LLVMFooDispose` ones.